keyboards/keycapsss/kimiko/rev2/info.json

```diff
@@ -8,6 +8,21 @@
         "pid": "0x4B69",
         "device_version": "2.0.0"
     },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "F4", "pin_b": "F5"}
+        ]
+    },
+    "split": {
+        "soft_serial_pin": "D2",
+        "encoder": {
+            "right": {
```

Comment:
Where do you found information about that.
There is not information about that here. https://docs.qmk.fm/#/reference_info_json?id=encoders

Reply:
Also seen thisa in the big PR 19923 "Move encoder config to data driven"
https://github.com/qmk/qmk_firmware/pull/19923/files
e.g. for keyboards/aleblazer/zodiark/info.json